src/debug-layer/debug-command-encoder.cpp

```
@@ -597,13 +597,151 @@ void DebugRayTracingPassEncoder::end()
+    m_commandEncoder->requireWorkGraphPass();
+
+    if (!m_pipelineBound)
+    {
+        RHI_VALIDATION_ERROR("No pipeline bound.");
+        return;
+    }
+    if (!backingStore)
+    {
+        RHI_VALIDATION_ERROR("'backingStore' must not be null.");
+        return;
+    }
```

Cppcheck (2.20.0)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)